src/auth-service/controllers/group.controller.js

```
@@ -829,6 +829,930 @@
+            .find({
+              targetId: grp_id,
+              requestType: "group",
+              status: "pending",
+            })
+            .populate("user_id", "firstName lastName email profilePicture")
+            .sort({ createdAt: -1 })
+            .lean();
+
+          result = {
+            success: true,
+            message: "Access requests retrieved successfully",
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)